script/install-knitr

```diff
@@ -19,7 +19,6 @@ ensure_r_installed() {
   elif [[ "${TRAVIS_OS_NAME:-}" == "osx" ]]; then
     echo "Installing R..."
     brew update
-    brew tap homebrew/science
```

Comment:
Why is this being removed in this PR? Is this causing errors during CI?

Reply:
Yes, homebrew/science is deprecated. R appears to be available without the tap.